### PR TITLE
Tower Forge -> Batch Forge

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Closes https://github.com/seqeralabs/docs/issues/13

### CODE REVIEW

File: platform_versioned_docs/version-23.1.0/compute-envs/azure-batch.mdx [hunk 1]
```diff
@@ -18,13 +18,13 @@ This guide assumes you have an existing [Azure account](https://azure.microsoft.
 
 There are two ways to create an Azure Batch compute environment in Tower:
 
-- [**Tower Forge**](#tower-forge): This option automatically creates the Azure Batch resources needed for your Tower compute environment. This eliminates the need to set up your Azure Batch infrastructure manually.
+- [**Batch Forge**](#tower-forge): This option automatically creates the Azure Batch resources needed for your Tower compute environment. This eliminates the need to set up your Azure Batch infrastructure manually.
 
 - [**Manual**](#manual): This option allows Tower to use existing Azure Batch resources.
```
Comment (ewels) [inline platform_versioned_docs/version-23.1.0/compute-envs/azure-batch.mdx:23]: Spotted another mention of Tower 😉 

```suggestion
- [**Manual**](#manual): This option allows Seqera Platform to use existing Azure Batch resources.
```

Comment (llewellyn-sl) [reply platform_versioned_docs/version-23.1.0/compute-envs/azure-batch.mdx:23]: That's v23.1 — pre-rebrand.

File: platform_versioned_docs/version-22.4.0/compute-envs/azure-batch.mdx [hunk 2]
```diff
@@ -20,15 +20,15 @@ This guide assumes you have an existing [Azure Account](https://azure.microsoft.
 
 There are two ways to create a **Compute Environment** for **Azure Batch** with Tower.
 
-1. **Tower Forge**: This option automatically manages the Azure Batch resources in your Azure account.
+1. **Batch Forge**: This option automatically manages the Azure Batch resources in your Azure account.
 
 2. **Manual**: This option allows you to create a compute environment using existing Azure Batch resources.
 
-If you don't yet have an Azure Batch environment fully set up, it is suggested that you follow the [Tower Forge](#tower-forge) guide.
+If you don't yet have an Azure Batch environment fully set up, it is suggested that you follow the [Batch Forge](#tower-forge) guide.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-22.4.0/compute-envs/azure-batch.mdx:27]:
```suggestion
If you don't yet have an Azure Batch environment fully set up, follow the [Batch Forge](#tower-forge) guide to do so.
```

File: platform_versioned_docs/version-23.1.0/_todo.mdx [hunk 3]
```diff
@@ -17,13 +17,13 @@ title: "\_todo"
 - [Can I have Nextflow automatically retry Tasks that fail due to Spot instance reclamation?](#aws_spot_retry)
 - **Can I have Nextflow automatically retry Tasks that fail due to Spot instance reclamation?** <a id="aws_spot_retry"></a>
 
-  Yes. As of Tower version ?????, any Spot-based AWS Batch Compute Environment created by Tower Forge will be automatically configured to retry each process 3 times. ??? If a retry policy is not defined???
+  Yes. As of Tower version ?????, any Spot-based AWS Batch Compute Environment created by Batch Forge will be automatically configured to retry each process 3 times. ??? If a retry policy is not defined???
```
Comment (justinegeffen) [inline platform_versioned_docs/version-23.1.0/_todo.mdx:20]: Can we remove this or specify the version?

**Review by justinegeffen — APPROVED** [review]
Some minor feedback but otherwise this is great, thank you!

**Pull request merged**